code/game/machinery/doors/poddoor.dm

```diff
@@ -64,3 +64,36 @@
 /obj/machinery/door/poddoor/try_to_crowbar(obj/item/I, mob/user)
 	if(stat & NOPOWER)
 		open(1)
+
+/obj/machinery/door/poddoor/multi_tile // Whoever wrote the old code for multi-tile spesspod doors needs to burn in hell.
+	name = "Large Pod Door"
+
+/obj/machinery/door/poddoor/multi_tile/four_tile_ver/
+	icon = 'icons/obj/doors/1x4blast_vert.dmi'
+	bound_height = 128
+	dir = NORTH
+
+/obj/machinery/door/poddoor/multi_tile/three_tile_ver/
+	icon = 'icons/obj/doors/1x3blast_vert.dmi'
+	bound_height = 96
+	dir = NORTH
+
+/obj/machinery/door/poddoor/multi_tile/two_tile_ver/
+	icon = 'icons/obj/doors/1x2blast_vert.dmi'
+	bound_height = 64

```

Comment:
This breaks a number of checks, so you can't do it.

Reply:
@MoreRobustThanYou